plasma/cli/client.py

```diff
@@ -20,12 +20,8 @@ def start_client_cmd(ctx):
 
 class ClientParser():
 
-    def __init__(self):
-        self.client = Client()
-        self.db = plyvel.DB('/tmp/plasma_mvp_db/', create_if_missing=True)
-        self.current_block = self.client.get_current_block_num()
-        self.synced_block = 1
-        self.client_cmds = dict(
+    synced_block = 1
+    client_cmds = dict(
```

Comment:
Moving `client_cmds` to a class variable means we don't have access to `self` so we need to change to:
```
    client_cmds = dict(
        sync=ClientParser.sync_child_chain,
        deposit=ClientParser.deposit,
        send_tx=ClientParser.send_tx,
        submit_block=ClientParser.submit_block,
        withdraw=ClientParser.withdraw,
        help=ClientParser.help,
    )
```